main.py

```diff
@@ -3,22 +3,25 @@
 Either create a Flask run configuration on this module or set up to run it locally with main.
 
 """
-
+from plato.domain import StorageType
 from plato.flask_app import create_app
-from plato.settings import WORKING_DB_URL, PROJECT_NAME, PROJECT_VERSION, TEMPLATE_DIRECTORY, S3_TEMPLATE_DIR, S3_BUCKET
-from plato.util.setup_util import create_template_environment, setup_swagger_ui, load_templates
+from plato.settings import WORKING_DB_URL, PROJECT_NAME, PROJECT_VERSION, TEMPLATE_DIRECTORY, STORAGE_TYPE, \
+    TEMPLATE_DIRECTORY_NAME
+from plato.util.setup_util import create_template_environment, setup_swagger_ui, initialize_file_storage
 
 template_environment = create_template_environment(TEMPLATE_DIRECTORY)
 swagger_ui_config = setup_swagger_ui(PROJECT_NAME, PROJECT_VERSION)
+file_storage = initialize_file_storage(STORAGE_TYPE)
+
 app = create_app(db_url=WORKING_DB_URL,
                  template_static_directory=f"{TEMPLATE_DIRECTORY}/static",
                  jinja_env=template_environment,
-                 swagger_ui_config=swagger_ui_config
-                 )
+                 swagger_ui_config=swagger_ui_config,
+                 storage=file_storage)
 
 if __name__ == '__main__':
     # in app-context setups
     with app.app_context():
-        load_templates(S3_BUCKET, TEMPLATE_DIRECTORY, S3_TEMPLATE_DIR)
+        if STORAGE_TYPE == StorageType.S3:
```

Comment:
So I assume that if we are using local storage, we will directly use that folder as the templates folder, so no need to pre-load them?

Reply:
yep! I thought about moving this  piece of code to the initialize file storage object, but we need the app to be created at that point...so here it stayed!